frontend/src/metabase/reference/databases/databases.integ.spec.js

```diff
@@ -0,0 +1,156 @@
+import TableQuestionsContainer from "metabase/reference/databases/TableQuestionsContainer";
+import FieldListContainer from "metabase/reference/databases/FieldListContainer";
+import FieldDetailContainer from "metabase/reference/databases/FieldDetailContainer";
+
+import DatabaseList from "metabase/reference/databases/DatabaseList";
+import List from "metabase/components/List.jsx";
+import ListItem from "metabase/components/ListItem.jsx";
+import ReferenceHeader from "../components/ReferenceHeader.jsx";
+import AdminAwareEmptyState from "metabase/components/AdminAwareEmptyState.jsx";
+
+describe("The Reference Section", () => {
+    // Test data
+    const cardDef = { name :"A card", display: "scalar", 
+                      dataset_query: {database: 1, table_id: 1, type: "query", query: {source_table: 1, "aggregation": ["count"]}},
+                      visualization_settings: {}}
+
+    // Scaffolding
+    beforeAll(async () => {
+        await login();
+
+    })
+
+    describe("The Data Reference for the Sample Database", async () => {
+        
+        // database list
+        it("should see a single database", async ()=>{
+            const store = await createTestStore()
+            store.pushPath("/reference/databases/");
+            var container = mount(store.connectContainer(<DatabaseListContainer />));
+            await store.waitForActions([FETCH_DATABASES, END_LOADING])
+            
+            expect(container.find(ReferenceHeader).length).toBe(1)
+            expect(container.find(DatabaseList).length).toBe(1)            
+            expect(container.find(AdminAwareEmptyState).length).toBe(0)
+            
+            expect(container.find(List).length).toBe(1)
+            expect(container.find(ListItem).length).toBe(1)
+        })
+        
+        // database detail
+        it("should see a the detail view for the sample database", async ()=>{
+            const store = await createTestStore()
+            store.pushPath("/reference/databases/1");
+            mount(store.connectContainer(<DatabaseDetailContainer />));
+            await store.waitForActions([FETCH_DATABASE_METADATA])
+
+        })
+        
+        // table list
+       it("should see the 4 tables in the sample database",async  () => {
+            const store = await createTestStore()
+            store.pushPath("/reference/databases/1/tables");
+            mount(store.connectContainer(<TableListContainer />));
+            await store.waitForActions([FETCH_DATABASE_METADATA])
+
+            expect(4).toBe(4);
+        })
+        // table detail
+
+       it("should see the Orders table", async  () => {
+            const store = await createTestStore()
+            store.pushPath("/reference/databases/1/tables/1");
+            mount(store.connectContainer(<TableDetailContainer />));
+            await store.waitForActions([FETCH_DATABASE_METADATA])
+            expect(true).toBe(true);
```

Comment:
`expect(true).toBe(true);` is unnecessary – the test will pass if the code doesn't produce errors